packages/shared/src/utils/cfd/available-cfd-accounts.ts

```diff
@@ -22,7 +22,6 @@ export const getCFDAvailableAccount = () => [
         market_type: 'synthetic',
         product: 'standard',
         icon: 'Standard',
-        availability: 'Non-EU',
     },
     {
         name: 'Deriv X',
```

Comment:
Is Deriv X availability kept as it was ? Since we do not have any API call for Deriv X availability, we need to send a request for it. It does not make sense to set the availability of cTrader and MT5 dynamically but DerivX statically.
cc: @matin-deriv @farrah-deriv

Reply:
we have a card for it and the BE team are yet to implement this. I think they are planning to start after the defaulting jurisdiction task.

Reply:
They are referring to `trading_platform_product` then.